.github/workflows/deploy.yml

```diff
@@ -0,0 +1,92 @@
+name: Trigger Target Workflow
+
+on:
+  workflow_dispatch
```

Comment:
so we are gonna manually start this workflow?

Reply:
The workflow will automatically be triggered when there is a new tag made for the redcap data dictionary. Here is the actions file I plan on sending to TDOM. It will send a single to trigger this workflow here: https://github.com/kind-lab/bridge2ai-redcap/blob/main/.github/workflows/deploy.yml

Reply:
@yibeichan but you're right in that, with this trigger type, you will also be able to manually dispatch it from the GitHub UI